grakn-engine/src/main/java/ai/grakn/engine/session/GraqlSession.java

```diff
@@ -87,7 +88,7 @@
     private List<Query<?>> queries = null;
 
     GraqlSession(
-            Session session, GraknGraphFactory factory, String outputFormat,
+            Session session, GraknSession factory, String outputFormat,
```

Comment:
Again, I don't think you should do it right now, better to get the PR in and find out the actual problems. But eventually that 'factory' nomenclature will have to be changed...